src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -57,4 +57,29 @@
     </EmbeddedResource>
   </ItemGroup>
 
+  <PropertyGroup>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);GatherNativeFilesForTfm</TargetsForTfmSpecificContentInPackage>
+  </PropertyGroup>
+
+  <Target Name="GatherNativeFilesForTfm">
+    <ItemGroup>
+      <!-- Create profiler library items for all of the native platforms. -->
+      <MonitorProfilerLibraryFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)MonitorProfiler%(LibraryExtension)')">
+        <PackagePath>tools/$(TargetFramework)/any/profiler/%(TargetRid)</PackagePath>
+      </MonitorProfilerLibraryFile>
+    </ItemGroup>
+    <ItemGroup>
+      <!-- Create profiler symbols items for all of the native platforms. -->
+      <MonitorProfilerSymbolsFile Include="@(NativeArtifactDirectories->'%(Identity)\%(LibraryPrefix)MonitorProfiler%(SymbolsExtension)')">
+        <PackagePath>tools/$(TargetFramework)/any/profiler/%(TargetRid)</PackagePath>
+      </MonitorProfilerSymbolsFile>
+    </ItemGroup>
+    <ItemGroup>
+      <!-- Pack the profiler library for each platform if it exists. -->
+      <TfmSpecificPackageFile Include="@(MonitorProfilerLibraryFile->Exists())" />
```

Comment:
Isn't this going it yield something like `true;true;true;true;false;true`?

Reply:
It's a filter, not a transform: https://docs.microsoft.com/en-us/visualstudio/msbuild/item-functions?view=vs-2022